### PR TITLE
fix: use `onChildrenLoaded` props to filter orgUnits with coordinates

### PR DESCRIPTION
**Pull request by delemaf**
use default app namespace### :pushpin: References

-   **Issue:** [Closes](https://app.clickup.com/t/86957kfzj) #86957kfzj

### :memo: Implementation
⚠️ Require PR https://github.com/EyeSeeTea/d2-ui-components/pull/241
- OrgUnits with coordinate are not all fetched anymore but fetch on OrgUnitsSelector change
- Filtering by geometry props if OrgUnitsSelector items have a checkbox

### :art: Screenshots
https://github.com/user-attachments/assets/8ddb3340-77f3-47ce-9ffe-e992db50e9ad

### CODE REVIEW

**Comment by ifoche** [comment]
Task linked: [CU-86957kfzj Filter out OU before asking for all points in "import rule"](https://app.clickup.com/t/86957kfzj)

File: src/webapp/components/org-unit/WithCoordinatesOrgUnitsSelector.tsx [hunk 1]
```diff
@@ -17,20 +18,23 @@ const WithCoordinatesOrgUnitsSelector: React.FC<OUDialogProps> = ({
...
 
     const controls = {
         filterByLevel: true,
         filterByGroup: true,
         selectAll: true,
     };
 
+    const onFilterWithCoordinates = useCallback(
+        (orgUnits: OrgUnit[]) => {
+            setOrgUnitsWithCoordinates([
```
Comment (tokland) [inline src/webapp/components/org-unit/WithCoordinatesOrgUnitsSelector.tsx]: To avoid race conditions, we can use `setOrgUnitsWithCoordinates(prev => ...)`

**Review by tokland — COMMENTED** [review]
[code only review]

**Review by tokland — APPROVED** [review]
[code only review]

**Review by MiquelAdell — APPROVED** [review]

**Pull request merged**